tracer/src/Datadog.Trace.Bundle/Datadog.Trace.Bundle.targets

```diff
@@ -0,0 +1,77 @@
+         see https://github.com/dotnet/sdk/blob/67f43d9366503d46a1063319a93a388b705ae8a5/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets#L504
+         The following lines remove the files we DON'T want published from ResolvedFileToPublish -->
+    <ItemGroup>
+      <!-- .NET Framework: always remove native files in "linux-*" -->
+      <DatadogFilesToRemove Include="@(ResolvedFileToPublish)"
+                            Condition="$(IsDotNetFramework) AND
+                                       $([System.String]::Copy('%(ResolvedFileToPublish.RelativePath)').ToLower().StartsWith('datadog\linux-'))" />
+      <!-- .NET Framework: always remove native files in "osx*"
+           note: currently there are two osx directories: "osx" and "osx-x64", so we can't match "osx-*" -->
+      <DatadogFilesToRemove Include="@(ResolvedFileToPublish)"
+                            Condition="$(IsDotNetFramework) AND
+                                       $([System.String]::Copy('%(ResolvedFileToPublish.RelativePath)').ToLower().StartsWith('datadog\osx'))" />
+      <!-- .NET Framework: if Platform == "x64", remove native files in "win-x86" -->
+      <DatadogFilesToRemove Include="@(ResolvedFileToPublish)"
+                            Condition="$(IsDotNetFramework) AND '$(Platform)' == 'x64' AND
+                                       $([System.String]::Copy('%(ResolvedFileToPublish.RelativePath)').ToLower().StartsWith('datadog\win-x86\'))" />
+      <!-- .NET Framework: if Platform == "x86", remove native files in "win-x64" -->
+      <DatadogFilesToRemove Include="@(ResolvedFileToPublish)"
+                            Condition="$(IsDotNetFramework) AND '$(Platform)' == 'x86' AND
+                                       $([System.String]::Copy('%(ResolvedFileToPublish.RelativePath)').ToLower().StartsWith('datadog\win-x64\'))" />
+
+      <!-- .NET Core: if $(RuntimeIdentifier) is set, remove the native library directories that don't match -->
+      <DatadogFilesToRemove Include="@(ResolvedFileToPublish)"
+                            Condition="!$(IsDotNetFramework) AND '$(RuntimeIdentifier)' != '' AND
+                                         ($([System.String]::Copy('%(ResolvedFileToPublish.RelativePath)').ToLower().StartsWith('datadog\win-')) OR
+                                          $([System.String]::Copy('%(ResolvedFileToPublish.RelativePath)').ToLower().StartsWith('datadog\linux-')) OR
+                                          $([System.String]::Copy('%(ResolvedFileToPublish.RelativePath)').ToLower().StartsWith('datadog\osx'))) AND
+                                       !$([System.String]::Copy('%(ResolvedFileToPublish.RelativePath)').ToLower().StartsWith('datadog\$(RuntimeIdentifier)'))" />
+
+      <!-- if $(EffectiveTargetFramework) is set, remove the managed library directories that don't match -->
+      <!-- TODO: should we throw error if unsupported TargetFramework? -->
```

Comment:
TODO: should we throw error if unsupported TargetFramework?

Reply:
IMO yes. And in the error, ask them to contact support (so we can understand the issue) but also mention that they can disable pruning to unblock themselves

Reply:
Done, thanks.